faf-java-server-app/src/main/java/com/faforever/server/entity/Player.java

```diff
@@ -65,13 +69,20 @@
   @Transient
   private ClientConnection clientConnection;
 
+  @Transient
+  private Map<Integer, FraudReport> fraudReportsByReporterId = new HashMap<>();
```

Comment:
So that just lies within the server and has no effect?

Reply:
This map is used, or at least its keys, However, the FraudReports it stores aren't. Initially, I thought I would store the spoofed values so I can compare them later but I realized that this wouldn't help.

Now we can either remove `FraudReport` and just have a counter of how many players reported fraud, or we could somehow make use of `FraudReport` (i.e. store it in the DB for moderator review).

What are your thoughts on this?

Reply:
I would store them for moderators. Is the only thing that makes sense to me. We can show them in the moderator client. Or you can issue a ban automaticly, but that is dangerous because ppl could modify clients to make false reports 🤔 So having a moderator decide is probably the best. Maybe also a message to the host that player xyz spoofed his/her data would be cool. But that would also be some extra worrk in game and server.

Reply:
I just say you actually tell all players to disconnect from players that spoofed rating

Reply:
Idk if saving the fraud reports is nedded then any longer 🤔 but would be good still maybe we ccan save them and only make an modor integration once it is really needed. Because spoofing will no longer make sense if everybody will disconnected from you if u do so.